packages/layouts-css/src/breakpoints.scss

```diff
@@ -1,31 +1,31 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @mixin mobile {
-  @media (max-width: 486px) {
+  @media (max-width: 31.25rem) { //500px
     @content;
   }
 }
 
 @mixin landscape-mobile {
-  @media (min-width: 487px) and (max-width: 768px) {
+  @media (max-width: 68.75rem) { //1100px
```

Comment:
`landscape-mobile` can be removed i think.

it will be a breaking change but I think we are ok with that.

in all the places that it is currently used, we will need to use something else

Reply:
Ok so I removed landscape-mobile in the scss files where it was referenced/used, and I think anything that used it before will now just default to table because it's within the same range. However I did not touch the typescript stuff, seems sketchy, it is referenced a lot. Like `GridItem.tsx` or in `GridDemo.tsx`. Should we have those removed as well? 🤔

Reply:
yes i think the TS stuff will need to be changed as well, since we are not going to offer some of those breakpoints anymore.

Reply:
I removed landscape mobile completely from the code. Also updated some texts or comments regarding breakpoints to new values